### DISCUSSION TITLE
The course never progresses past creating a repository

### DISCUSSION THREAD

**Discussion by leebut** [skills/.github#554] [Code with Codespaces]
Hello,

This exercise does not progress past the create repository part. The README is refreshes to the Welcome README, and no new instructions are given.

I followed the steps in the Welcome README.
* I opened the exercise in a new tab
* created a new repository with the filled in details by opening the **Start Course** link in a new tab, with the filled in details
* refreshed the page after 20 seconds
* The welcome README is displayed again.

I tried deleting the new repository and starting again, with the same result.

**Comment by sinsukehlab** [comment]
Will be resolved in skills/code-with-codespaces#86
